redirects.js

```diff
@@ -609,7 +609,7 @@ const redirects = [
   },
   {
     from: "/articles/install/multi-agent",
-    to: "/articles/environment-management",
+    to: "/articles/agents-overview",
```

Comment:
i don't think we can do this since we also have an /articles/environment-management page which people need to be able to get to..